src/runtime.js

```diff
@@ -7,10 +7,11 @@ var Obj = require('./object');
 // we know how to access variables. Block tags can introduce special
 // variables, for example.
 var Frame = Obj.extend({
-    init: function(parent) {
+    init: function(parent, inMacro) {
```

Comment:
Though since it's internal, we can also leave it as-is for now and rename it later if necessary; I'm fine with that if you prefer.